Complete Attraction and MRT Station APIs and deploy your website to AWS EC2 instance

youyuhsuan opened the pull request:
API Endpoints:
- Get all MRT station information: "/api/mrts"
- Get specific attraction information: "/api/attraction/{attractionId}"
- Get all attractions information: "/api/attractions"

Deployed website link: [taipei-attractions](http://54.64.137.35:8000/docs)

cwpeng reviewed (changes requested):
- Response format of http://54.64.137.35:8000/api/mrts is incorrect,
- Response format of http://54.64.137.35:8000/api/attraction/10 is incorrect.
- images field of http://54.64.137.35:8000/api/attraction/10 should be an array containing image URLs for each attraction. 
- Naming of nextPage is incorrect.

youyuhsuan commented:
- Changed 'dic' to 'array'.
- Replaced 'cursor.fetchall()' with 'cursor.fetchone()'.
- Changed 'images field' to 'array'.
- Renamed 'nextpage' to 'nextPage'."

cwpeng reviewed (approved):
OK. Done.
